jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryAuthenticator.java

```diff
@@ -86,7 +86,7 @@ public RegistryAuthenticator initialize()
         throw new RegistryAuthenticationFailedException(ex);
 
       } catch (InsecureRegistryException ex) {
-        // HTTP is not allowed, so just return null.
+        // Cannot skip certificate validation or use HTTP, so just return null.
```

Comment:
This doesn't seem related.

Reply:
I think it's small enough that it's fine to keep it in the PR

Reply:
No worries. Will have this in a later PR.